public/js/timeline.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
```

Comment:
Interestingly enough, public/js/ is already in the gitignore.
I can add the file directly, ofc, but shouldn't it just be ignored in the first place?